packages/@react-spectrum/s2/stories/Tabs.stories.tsx

```diff
@@ -25,9 +27,9 @@ const meta: Meta<typeof Tabs> = {
 export default meta;
 
 export const Example = (args: any) => (
-  <Tabs {...args}>
+  <Tabs {...args} styles={style({maxWidth: '[450px]', height: 144})}>
```

Comment:
```suggestion
  <Tabs {...args} styles={style({width: '[450px]', height: 144})}>
```
This way changing tabs won't change the size overall